lib/hyrax/configuration.rb

```diff
@@ -417,9 +417,7 @@ def uploader
 
     attr_writer :fedora_schema
     def fedora_schema
-      @fedora_schema ||= {
-        depositor: RDF::URI("http://id.loc.gov/vocabulary/relators/dpt")
-      }
+      @fedora_schema ||= Hyrax::FedoraSchema.fedora_schema
```

Comment:
as this is already a class attribute. I see no need to memoize this any longer.

Reply:
Could just be `delegate :fedora_schema, :fedora_schema=, to: Hyrax::FedoraSchema`

Reply:
that won't mess up overriding it locally will it?

Reply:
Done. Learning new things every day ... today 'memoize'!